### PR TITLE
database-assignment-1-lhphuc

### PR DESCRIPTION
**Pull request by hoangphuc552001**

### CODE REVIEW

File: system.sql [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    FOREIGN KEY (room_id) REFERENCES Room (room_id)
+    );
+
+alter table Room
+    add FOREIGN KEY (class_id) REFERENCES Class (class_id);
+
+
+create table if not exists Student
+(
+    stud_id     int primary key,
```
Comment (tpdongcs) [inline system.sql]: primary key usually use auto increase instead.

File: system.sql [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    stud_id  int,
+    class_id int,
+    grade    varchar(3),
+    primary key (stud_id, class_id),
+    FOREIGN KEY (stud_id) REFERENCES Student (stud_id),
+    FOREIGN KEY (class_id) REFERENCES Class (class_id)
+    );
+
+INSERT INTO Professor (prof_id, prof_lname, prof_fname)
+VALUES (1, 'Smith', 'John'),
```
Comment (tpdongcs) [inline system.sql]: and we dont need to indicate primary key when insert like this

File: system.sql [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+SELECT distinct(CONCAT(s.stud_fname, ' ', s.stud_lname)) AS 'Student Name',
+        AvgGrade.Grade                            AS 'Average Grade',
+        CASE
+            WHEN AvgGrade.Grade < 5 THEN 'Weak'
+            WHEN AvgGrade.Grade >= 5 AND AvgGrade.Grade < 8 THEN 'Average'
+            WHEN AvgGrade.Grade >= 8 THEN 'Good'
+            END                                   AS 'Grade'
+FROM Student s
+         JOIN Enroll e ON s.stud_id = e.stud_id
+         JOIN (SELECT e.stud_id,
```
Comment (tpdongcs) [inline system.sql]: can you find a way not to join 3 table like this , and avoid joining with a subquery.

**Comment by hoangphuc552001** [comment]
I just modified my code follow your instruction

**Pull request merged**